frontend/scss/app/components/playlist/_playlist_update_view.scss

```
@@ -27,4 +27,28 @@
+    line-height: 1.35;
+    padding: var(--gap-default);
+    border-radius: var(--border-radius);
+    border: 1px solid var(--border-color);
+    -webkit-overflow-scrolling: touch;
+    touch-action: pan-y;
+    overscroll-behavior: contain;
+  }
+
+  &__log-line {
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)